_pages/dev/extending-checkout.md

```diff
@@ -394,6 +394,51 @@ provideConfig({
 
 **Note:** You can use this same approach to combine all the steps into a single-step checkout.
 
+## Multiple Checkout Flows
+
+Starting from version [x.x] a new checkout property named "flows" has been introduced. This enhancement facilitates greater flexibility and dynamism in customizing the checkout process based on specific criteria returned from the backend.
+
```

Comment:
Is a specific version of the back end required for this work? And if so, do we know which version that is?

Reply:
There is no any specific backend version required for it.